INSTALL.md

```diff
@@ -204,10 +204,10 @@ make
 By default, the executable that is produced is not ready for distribution. Install https://github.com/auriamg/macdylibbundler
 
 ```bash
-dylibbundler -od -b -x ./bin/SuperTuxKart.app/Contents/MacOS/supertuxkart -d ./bin/SuperTuxKart.app/Contents/libs/ -p @executable_path/../libs/
+dylibbundler -od -b -x ./bin/SuperTuxKart.app/Contents/MacOS/supertuxkart -d ./bin/SuperTuxKart.app/Contents/libs/ -p /usr/local/opt/
```

Comment:
This is wrong, this change will make the executable impossible to distribute

Reply:
Fixed.